kobo/apps/languages/views/translation_service.py

```diff
@@ -1,64 +1,60 @@
 # coding: utf-8
-from drf_spectacular.utils import extend_schema
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import extend_schema, extend_schema_view, OpenApiParameter
+from rest_framework.renderers import JSONRenderer
 
+from kpi.utils.schema_extensions.markdown import read_md
+from kpi.utils.schema_extensions.response import open_api_200_ok_response
 from ..models.translation import TranslationService
 from ..serializers import TranslationServiceSerializer
 from .base import BaseViewSet
 
 
 @extend_schema(
-    tags=['translation-services'],
+    tags=['Translation Services'],
+)
+@extend_schema_view(
+    list=extend_schema(
```

Comment:
Missing documentation for 400 error on a GET request. 

<img width="1126" height="660" alt="Image" src="https://github.com/user-attachments/assets/f65ed6a6-fe95-409a-8f02-97d3023bfaf0" />